release.config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is used with https://github.com/MetaMask/create-release-branch to determine our release strategy